src/components/annual-report/report-data/SectionOne.js

```diff
@@ -77,64 +87,300 @@ const reportQueries = {
     },
 }
 
-export const SectionOne = () => {
+export const SectionOne = ({ reportParameters, previewReport }) => {
     const { data, loading, error, refetch } = useDataQuery(reportQueries, {
         lazy: true,
     })
 
-    const generateReport = () => {
-        const periods = ['2022', '2021', '2020', '2020']
-        const currentPeriod = periods[0]
-        // const dataSets = ['YmRjo8j3F3M']
-        const dataSets = [
-            'dONyxVsQyGS',
-            'rGDF7yDdhnj',
-            'YmRjo8j3F3M',
-            'GhdP8W2GorO',
-        ]
-        // const dataElements = ['bcDTj5odXAg']
-        const dataElements = [
-            'ieThL7l107F',
-            'RvArfQFKdXe',
-            'XeRBhx8avQY',
-            'YRJjIr5tuD6',
-            'YAAmrY2RPbZ',
-            'DvUwQScvSpc',
-        ]
-        const orgUnits = ['lZsCb6y0KDX']
-        const orgUnitLevel = 'LEVEL-2'
+    let sectionData = null
 
-        const variables = {
-            dataSets,
-            dataElements,
-            orgUnits,
-            orgUnitLevel,
-            periods,
-            currentPeriod,
-        }
-        refetch(variables)
+    // const generateReport = () => {
+    const periods = reportParameters.periods
+    const currentPeriod = reportParameters.currentPeriod
+    // const dataSets = ['YmRjo8j3F3M']
+    const dataSets = reportParameters.dataSets
+    const dataElements = reportParameters.dataElements
+    const orgUnits = reportParameters.orgUnits
+    const orgUnitLevel = reportParameters.orgUnits
+
+    const variables = {
+        dataSets,
```

Comment:
you can just define this like `const variables = {...reportParameters}` and then overwrite any values that differ from the reportParameters defaults